src/Microsoft.VisualStudio.Editors/PropPages/ApplicationPropPage.vb

```diff
@@ -676,6 +676,8 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
 
             'Populate the target framework combobox
             PopulateTargetFrameworkComboBox(Me.TargetFramework)
+
+            Me.AssemblyInfoButton.Visible = Not Utils.IsTargetingDotNetCore(Me.ProjectHierarchy)
```

Comment:
I agree and looking at the code I think I found one other bug - The package page will show up for all projects right now so first we need to add a mechanism to check for this. With that in mind, the property pages have no knowledge of CPS right now so they wouldn't be able to do capabilities. I think we need to introduce a property (that can be backed by a capability) which then the property pages can consume.